src/features/user/MFV2/InfoAndCTA/DonorCircleInvitation/SingleInvitationCard.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styles from '../InfoAndCta.module.scss';
+
+interface Props {
+  link: string;
+  image: string;
+  title: string;
+  subtext: string;
+}
+
+const SingleInvitationCard = ({ link, image, title, subtext }: Props) => {
+  return (
+    <a
+      className={styles.singleInvitationCardContainer}
+      href={link}
+      target="_blank"
+      rel="noreferrer"
+    >
+      <img src={image} alt={title} />
+      <h3 className={styles.singleInvitationCardTitle}>{title}</h3>
+      <p className={styles.singleInvitationCardText}>{subtext}</p>
+    </a>
+  );
+};
```

Comment:
1. Rename as per the comments for the DonorCircleInvitation component.
2. UI - the text alignment does not match the figma design. Was this to be changed?

Reply:
Text here is aligned to the extreme left of the card (in line with the left of the image), but in the design there seems to be some padding there.